src/main/java/uk/gov/hmcts/darts/event/service/impl/EventMappingServiceImpl.java

```diff
@@ -94,4 +119,8 @@ private EventMapping mapToEventMapping(EventHandlerEntity eventEntity) {
 
         return mapping;
     }
+
+    private boolean isUpdateToExistingMappingRequest(Boolean isRevision) {
+        return isRevision != null && isRevision;
+    }
```

Comment:
I think we can eliminate this method if the comment on `src/main/resources/openapi/event.yaml` can be incorporated.